style_guide.md

```diff
@@ -119,26 +120,26 @@ This is a list of words/abbreviations that are used inconsistently at the moment
 
 Use the following common contractions:
 
```

Comment:
Comment: I turned this into a list because it wasn't being displayed properly, some elements were shown on the same line.